core/src/com/mygdx/game/MyGdxGame.java

```diff
@@ -49,41 +42,38 @@
         return new ExtendViewport(300, 300, camera);
     }
 
-    private @NonNull ModelInstance createModelInstance() {
-        Model model = new ModelBuilder().createBox(25, 25, 25,
+    private @NonNull ModelInstance createDebugBoxModelInstance() {
+        var model = new ModelBuilder().createBox(25, 25, 25,
```

Comment:
sam czasami nie jestem przekonany do tych varów, 

jak są pętle for(int i = ) i tam dawałem vara to trochę bardzo na wyrost